app/assets/stylesheets/active_admin/_base.scss

```diff
@@ -4,38 +4,45 @@
 
 // Partials
 @import "./typography";
-@import "./header";
-@import "./forms";
-@import "./components/comments";
-@import "./components/flash_messages";
-@import "./components/date_picker";
-@import "./components/tables";
-@import "./components/batch_actions";
-@import "./components/modal_dialog";
-@import "./components/blank_slates";
-@import "./components/breadcrumbs";
-@import "./components/dropdown_menu";
-@import "./components/buttons";
-@import "./components/grid";
-@import "./components/links";
-@import "./components/pagination";
-@import "./components/panels";
-@import "./components/columns";
-@import "./components/scopes";
-@import "./components/status_tags";
-@import "./components/table_tools";
-@import "./components/index_list";
-@import "./components/unsupported_browser";
-@import "./components/tabs";
-@import "./pages/logged_out";
-@import "./structure/footer";
-@import "./structure/main_structure";
-@import "./structure/title_bar";
 
-body {
-  @include sans-family;
-  line-height: 1.5;
-  font-size: 72%;
-  background: $body-background-color;
-  color: $text-color;
+@media screen {
+  @import "./header";
+  @import "./forms";
+  @import "./components/comments";
+  @import "./components/flash_messages";
+  @import "./components/date_picker";
+  @import "./components/tables";
+  @import "./components/batch_actions";
+  @import "./components/modal_dialog";
+  @import "./components/blank_slates";
+  @import "./components/breadcrumbs";
+  @import "./components/dropdown_menu";
+  @import "./components/buttons";
+  @import "./components/grid";
+  @import "./components/links";
+  @import "./components/pagination";
+  @import "./components/panels";
+  @import "./components/columns";
+  @import "./components/scopes";
+  @import "./components/status_tags";
+  @import "./components/table_tools";
+  @import "./components/index_list";
+  @import "./components/unsupported_browser";
+  @import "./components/tabs";
+  @import "./pages/logged_out";
+  @import "./structure/footer";
+  @import "./structure/main_structure";
+  @import "./structure/title_bar";
+
+  body {
+    @include sans-family;
+    line-height: 1.5;
+    font-size: 72%;
+    background: $body-background-color;
+    color: $text-color;
+  }
+}
+
+@media print {
```

Comment:
I had this line defined in the `_print.scss` partial instead. I'm not sure but I figured that is required for it to work because I think Sass only allows `@import "...";` at the outermost level.

Reply:
Yes, I moved it ouside, because since I was wrapping the rest of the styles under `@media screen`, moving it outside brought a nice simmetry in my opinion.

Not sure about Sass an not outermost imports, it seems to work just fine.

Reply:
Ok great, thank you!